python/src/spark_rapids_ml/params.py

```diff
@@ -338,6 +365,27 @@ def _infer_num_workers(self) -> int:
 
         return num_workers
 
+    def _get_cuml_param(self, spark_param: str, silent: bool = True) -> Optional[str]:
```

Comment:
Why factor this out?  It was only called in one place.

Reply:
I can _maybe_ see a case for symmetry of get/set, but I'm not sure we'd use `_get_cuml_param` anywhere else.

Reply:
Actually, itt's needed in my followup PR. :)